code/modules/jobs/job_types/agent.dm

```diff
@@ -94,7 +94,6 @@
 /datum/outfit/job/agent
 	name = "Agent"
 	jobtype = /datum/job/agent
-
```

Comment:
As funny as it is, can you revert this change?

Reply:
Can do,